### ISSUE TITLE
diffuse "roughness" slider like VrayMtl

### ISSUE THREAD

**Issue by sharktacos** [vidarn/ThunderLoom#12]
@petermcevoy @vidarn 

Gentlemen, a feature suggestion:
The VrayMtl has a diffuse "roughness" slider which goes between Lambertian diffuse (roughness=0) and Oren-Nayer diffuse (roughness=1). This has a "powdery" look to the diffuse shading which  looks nicer with soft fabrics like cotton. 

If this would be something that could be pulled from the VrayMtl BRDF into the Thunderloom shader (and therefore hopefully not hard to implement), I think it could visually be quite nice. Hope you will consider it.

best,
Derek

**Comment by vidarn** [comment]
Hi Derek,

This sounds like a good idea! I agree that such a slider would be useful, thanks for the suggestion. Right now we're working on getting thunderloom to a point where we can release a version 1.0 of the shader, with the most essential features finished. I won't add this to the v1.0 milestone, since it is not strictly "essential", even though it would be nice to have. However, once version 1.0 is finished, this is definitely a good candidate for what to implement next. 

Best regards,
Vidar

**Comment by sharktacos** [comment]
@vidarn yes, that makes sense, thanks.
